.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Android APK Build
+
+on:
+  push:
+    branches:
+      - 'release/**' # Only trigger on branches starting with 'release/'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+
+    - name: Build APK
+      run: ./gradlew assembleRelease
+
+    - name: Upload APK
+      uses: actions/upload-artifact@v2
```

Comment:
Upgrade to ``actions/upload-artifact@v3`` for improved performance and maintainability.